src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -431,6 +431,22 @@ public static SymbolInfo GetSymbolInfo(this SemanticModel semanticModel, SelectO
         }
 
         /// <summary>
+        /// Gets the symbol information for the property of a sub-property pattern.
+        /// </summary>
+        public static SymbolInfo GetSymbolInfo(this SemanticModel semanticModel, SubPropertyPatternSyntax node, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
To clarify my position, I think that, given SubPropertyPatternSyntax node, one should pass SubPropertyPatternSyntax.Left to get SymbolInfo for the target property or a field. If SubPropertyPatternSyntax node itself is passed, SymbolInfo.None should be returned.